index.html

```diff
@@ -183,49 +223,50 @@ <h2 class="marketing-strategies__title">Marketing Strategies</h2>
           <li class="point"></li>
         </ul>
       </section>
-      <footer class="footer">
+
+      <footer class="footer" id="pricing">
+
         <div class="footer__wrapper">
           <div class="footer__logo">
-            <a href="#">
-              <img src="./assets/icons/logo.svg" alt="Logo" />
+            <a class="footer__logo-a" href="index.html">
+              <img src="./assets/icons/logo.svg" alt="Logo" class="footer__logo-img" />
               <img class="footer__logo-text" src="./assets/icons/Productly.svg" alt="LogoText" />
             </a>
           </div>
 
           <div class="footer__nav footer-first">
-            <p class="nav__title">Quick links</p>
-            <ul>
-              <li class="footer-link"><a href="#">About Us</a> </li>
-              <li class="footer-link"><a href="#">Blog</a></li>
-              <li class="footer-link"><a href="#">Contact</a></li>
-              <li class="footer-link"><a href="#">FAQ</a></li>
+            <p class="nav__title">Quick Links</p>
+            <ul class="footer__nav-list">
+              <li class="footer-link"><a class="footer-link-a" href="#">About Us</a> </li>
```

Comment:
.footer-link-a - tag name used in class name
probably should be like next one with footer-link-item class

Reply:
completed